src/Symfony/Bundle/SecurityBundle/Tests/Functional/UserPasswordEncoderCommandTest.php

```diff
@@ -71,11 +71,25 @@ public function testEncodePasswordNoConfigForGivenUserClass()
         $this->passwordEncoderCommandTester->execute(array(
             'command' => 'security:encode-password',
             'password' => 'password',
-            'user-class' => 'Wrong/User/Class',
-            'salt' => 'AZERTYUIOPOfghjklytrertyuiolnbcxdfghjkytrfghjk',
+            '--user-class' => 'Wrong/User/Class',
```

Comment:
Souldn't it be `Wrong\User\Class`? Also I'm wondering if the `Class` keyword in the last segment of the namespace is syntactically valid. Maybe you should change to something like `Foo\Bar\User` instead.